assets/css/main.css

```diff
@@ -2853,6 +2862,10 @@ button:disabled,
 #menu ul a.opener.active:before, #menu ul span.opener.active:before {
   transform: rotate(-180deg);
 }
+#menu ul #navbar-selected {
```

Comment:
I noticed the menu highlights did not carry over the first time I pushed, so I added the changes to the _menu.scss file.

Reply:
it looks like you may have created a branch from a main branch that's old. Do you know if that's what may have happened?

Reply:
I think that may be the case. Confirming I merged in the latest changes from the main branch. Next time I can start a new branch to keep things a bit cleaner too.